realm-transformer/src/main/groovy/io/realm/transformer/BytecodeModifier.groovy

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.transformer
+import javassist.*
+import javassist.expr.ExprEditor
+import javassist.expr.FieldAccess
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+/**
+ * This class encapsulates the bytecode manipulation code needed to transform model classes
+ * and the classes using them.
+ */
+class BytecodeModifier {
+
+    private static final Logger logger = LoggerFactory.getLogger('realm-logger')
+
+    /**
+     * Adds Realm specific accessors to a model class.
+     * All the declared fields will be associated with a getter and a setter.
+     *
+     * @param clazz the CtClass to add accessors to.
+     */
+    public static void addRealmAccessors(CtClass clazz) {
+        logger.info "  Realm: Adding accessors to ${clazz.simpleName}"
```

Comment:
the logger level might slow down the transformation, shouldn't the level be `verbose` or `debug`